node/pkg/watchers/evm/chain_config.go

```diff
@@ -104,7 +104,8 @@ var (
 		vaa.ChainIDXRPLEVM: {Finalized: true, Safe: true, EvmChainID: 1440000, PublicRPC: "https://rpc.xrplevm.org/", ContractAddr: "0xaBf89de706B583424328B54dD05a8fC986750Da8"},
 		// vaa.ChainIDPlasma: {Finalized: true, Safe: true, EvmChainID: _, PublicRPC: "", ContractAddr: ""},
 		vaa.ChainIDCreditCoin: {Finalized: true, Safe: true, EvmChainID: 102030, PublicRPC: "https://mainnet3.creditcoin.network", ContractAddr: "0xaBf89de706B583424328B54dD05a8fC986750Da8"},
-		// vaa.ChainIDMoca: Not in Mainnet yet
+		// vaa.ChainIDMoca: Not in Mainnet ""},
+		vaa.ChainIDMegaETH: {Finalized: true, Safe: true, EvmChainID: 4326, PublicRPC: "https://alpha.megaeth.com/rpc", ContractAddr: "0xaBf89de706B583424328B54dD05a8fC986750Da8"},
```

Comment:
Is this RPC live?
`./testBlockInfo.sh https://alpha.megaeth.com/rpc
RPC: https://alpha.megaeth.com/rpc
Latest Block: Error retrieving block
Safe Block: Error retrieving block
Finalized Block: Error retrieving block`

Reply:
You don't have to